amazon-kinesis-client-multilang/src/main/java/software/amazon/kinesis/multilang/messages/LogMessage.java

```diff
@@ -0,0 +1,62 @@
+package software.amazon.kinesis.multilang.messages;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.function.Function;
+
+@Getter
+@Setter
+@Slf4j
+public class LogMessage extends Message {
+    /**
+     * The name used for the action field in {@link Message}.
+     */
+    public static final String ACTION = "log";
+
+    /**
+     * The shard id that this processor is getting initialized for.
+     */
+    private String message;
+    private String logLevel = "info";
+
+
+    public LogMessage() {
+        this.message = "initialized";
+    }
+
+    /**
+     * Default constructor.
+     */
+    public LogMessage(String message) {
+        this.message = message;
+        log.info("Client logging: " + this.message);
+    }
+
+    public Function<String, Boolean> getLogger() {
```

Comment:
This looks to be returning only a boolean, not a Function<String, Boolean>. Also everything returns true. What is the benefit of returning a value

Reply:
It's returning a function

```
return (m) -> {
                     log.debug(m);
                     return true;
                 };
```

Not sure a better return type since `Function` requires one.